source/help/apps/desktop-guide.rst

```diff
@@ -40,108 +40,95 @@ Editing servers and teams
+| Show red badge on taskbar icon to indicate unread messages                    | Yes                       | Yes                       |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Flash taskbar icon when a new message is received                             | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Start app on login                                                            | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Hide menu bar                                                                 | Yes                       |                           | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Toggle window visibility when clicking on the tray icon                       | Yes                       |                           |                           |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Show icon on menu bar                                                         |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
+| Leave app running in notification center when application window is closed    |                           | Yes                       | Yes                       |
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+
 
 Menu Bar
 ---------------------------------------------------------------------
 
 The desktop app contains a menu bar with additional features and shortcuts to streamline your experience. 
 
-If the menu bar is hidden, you may use the ALT key to display the menu. To have the menu displayed at all times, go to the Settings page and uncheck the **Hide menu bar** setting.
+If the menu bar is hidden, you may use the ALT key to display the menu. To have the menu displayed at all times, go to the Settings page by pressing CTRL/CMD+COMMA and uncheck the **Hide menu bar** setting.
 
```

Comment:
> If the menu bar is hidden, you may use the ALT key ...

Use of the word *may* can be ambiguous, especially to people who are new to English. It can mean *might*, or *can*, or *have permission to*. It's best to use either *might* or *can* instead of *may*. In this particular case though, you could probably get rid of "you may" altogether, so that you have:

>If the menu bar is hidden, use the ALT key ...